cypress/support/pages/yves/ssp-service/ssp-service-repository.ts

```diff
@@ -0,0 +1,80 @@
+import { injectable } from 'inversify';
+
+// Import Cypress types
+type Chainable<Subject = any> = Cypress.Chainable<Subject>;
+type JQueryElement = JQuery<HTMLElement>;
+
+@injectable()
+export class SspServiceRepository {
+  getSspServiceTable(): Chainable<JQueryElement> {
```

Comment:
looks like a wrong usage of repositories. You duplicated the whole repository. 
Please check the example of what should be in this file, thanks 

https://github.com/spryker/cypress-tests/blob/master/cypress/support/pages/yves/ssp-inquiry/ssp-inquiry-repository.ts